spec/datetimeformat.html

```diff
@@ -323,7 +323,7 @@ <h1>PartitionDateTimePattern ( _dateTimeFormat_, _x_ )</h1>
           1. Else if _p_ is equal to `"fractionalSecondDigits"`, then
             1. If _fractionalSecondDigits_ is greater than 0, then
               1. Let _v_ be _tm_.[[millisecond]].
-              1. Let _v_ be _v_ × 10<sup> ( 3 - _fractionalSecondDigits_) </sup>.
+              1. Let _v_ be floor(_v_ × 10<sup> ( 3 - _fractionalSecondDigits_)) </sup>.
```

Comment:
I think you want,

`_v_` in | `_fractionalSecondDigits_` | `_v_` out
---|---|---|
123 | 3 | 123
123 | 2 | 12
123 | 1 | 1

right?  Then your math is still not right.  I think you want

```
Let _v_ be floor(_v_ × 10<sup>( _fractionalSecondDigits_ - 3 )</sup>).
```